docs/README-psp.md

```diff
@@ -2,20 +2,25 @@ PSP
 ======
 SDL port for the Sony PSP contributed by 
    Captian Lex 
+   Francisco Javier Trujillo Mata
+   Wouter Wijsman
+
 
 Credit to
    Marcus R.Brown,Jim Paris,Matthew H for the original SDL 1.2 for PSP
    Geecko for his PSP GU lib "Glib2d"
 
 Building
 --------
-To build SDL2 library for the PSP, make sure psp-config is in the path and run:
-   make -f Makefile.psp
+To build SDL2 library for the PSP, make sure you have the latest PSPDev status and run:
+```bash
+cmake -S. -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=$PSPDEV/psp/share/pspdev.cmake -DSDL_TEST=ON

```

Comment:
I would not recommend people to use ``-DSDL_Test=ON`` here. If that is used, ``make install`` does not work unfortunately.